llvm/test/CodeGen/AMDGPU/atomic-image-load.ll

```diff
@@ -0,0 +1,37 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -march=amdgcn -mcpu=gfx1010 < %s | FileCheck %s -check-prefix=GFX10
```

Comment:
use explicit -global-isel=0 when mixing run lines

Reply:
This isn't done?

Reply:
It wasn't. Now it is.